New Sample: Realistic Lighting and Shadows (#528)
Esri/arcgis-runtime-samples-java: scene/realistic-lighting-and-shadows/README.md

# Realistic lighting and shadows

Show realistic lighting and shadows for the time of day.

![Image of realistic lighting and shadows](RealisticLightingAndShadows.png)

## Use case

You can use realistic lighting to evaluate the shadow impact of buildings and utility infrastructure on the surrounding community. This could be useful for civil engineers and urban planners, or for events management assessing the impact of building shadows during an outdoor event.

## How to use the sample

> Select one of the three lighting options to show that lighting effect on the SceneView. Select a time of day from the slider (based on a 24hr clock) to show the lighting for that time of day in the SceneView.

Reviewer: in the "How to use the sample" we avoid directly referencing any API classes since the focus is on how a user interacts with a sample. I'd suggest the following:

```suggestion
Select one of the three available lighting options to display that lighting effect. Adjust the slider to show the lighting effect for a particular time of day. The 3D buildings will display shadows when "Sun light with shadows" is selected. 
```

## How it works

> To add realistic lighting and shadows to the SceneView:

Reviewer: during earlier README formats, we kept a sentence like this in to lead the reader into the steps. However a decision was reached last year to remove any introductory sentences and jump straight into the steps. So we can remove this sentence. 
```suggestion
```

1. Create an `ArcGISScene` and display it in a `SceneView`.
2. Create a `Calendar` to define the time of day.
3. Set the sun time to that calendar with `sceneView.setSunTime(calendar)`.

> 4. Set the lighting mode of the SceneView to **no light**, **light**, or **light and shadows** with `sceneView.setSunLighting(LightingMode)`.

Reviewer: since we use enums to define the lighting mode, we should refer to them as such here, along with referring to lighting mode as a class (since it's the first time we're referring to lighting mode) and SceneView as scene view (since we've already described it as a class in step 1, and can therefore refer to it in plain english after that first usage) e.g. 
```suggestion
4. Set the `LightingMode` of the scene view to `NO_LIGHT`, `LIGHT`, or `LIGHT_AND_SHADOWS` with `sceneView.setSunLighting(LightingMode)`.
```

Show realistic lighting and shadows for the time of day.

> ![Image of realistic lighting and shadows](RealisticLightingAndShadows.gif)

Reviewer 2: Sorry about the confusion @sclaridge ! We generally keep both the PNG and the GIF if there is one. The readme then links the PNG, and the SampleViewer will automatically show the GIF if there is one

Reviewer 2: I'm also noticing the GIF looks a little small, it's at 400x200 px I think, and should be 800x400
